### PR TITLE
Fix 'organize questions' item, remove 'create getting started guide'

### LINKED ISSUE #2753: Admin checklist should correctly check off the "Organize Questions" item
On Proto:

The new admin checklist feature is rad, but it's telling me I should create labels…

screen shot 2016-06-13 at 5 34 21 pm

…even though I already have a ton of them:

screen shot 2016-06-13 at 5 34 31 pm

I tried creating a new label to see if it would clear out the tip, but it didn't.

screen shot 2016-06-13 at 5 37 20 pm

### PR DESCRIPTION
**Pull request by tlrobinson**
Resolves #2753

@kdoh Still needs copy for "add a database" description: https://github.com/metabase/metabase/blob/06da8a8b7ba794f9b3e58b9eed1a63a8553f3af2/src/metabase/api/setup.clj#L107

### CODE REVIEW

**Comment by kdoh** [comment]
@tlrobinson Added!

**Comment by kdoh** [comment]
This look ok to everyone?

File: src/metabase/api/setup.clj [hunk 1]
```diff
@@ -151,13 +151,7 @@
       :description "Keep everyone on the same page by creating canonnical sets of filters anyone can use while asking questions."
       :link        "/admin/datamodel/database"
       :completed   has-segments?
-      :triggered   (>= num-cards 30)}
-     {:title       "Create a getting started guide"
-      :group       "Curate your data"
-      :description "Have a lot of data in Metabase? A getting started guide can help your team find their way around."
-      :completed   false                               ; TODO - how do we determine this?
-      :triggered   (and (>= num-cards 10)
-                        (>= num-users 5))}]))
```
Comment (salsakran) [inline src/metabase/api/setup.clj]: maybe this diff could be the starting point of a getting started PR?

Comment (tlrobinson) [inline src/metabase/api/setup.clj]: I think we need to merge this branch into master first.

**Pull request merged**